src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private int value;
+    private int bound = 100;
+
+    public String getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```

Reply:
If I do this variant, I have problem:
java: incompatible types: java.lang.String cannot be converted to core.basesyntax.Ball.
Help me to fix such mistake